packages/tldraw/src/lib/TldrawImage.tsx

```diff
@@ -186,7 +186,13 @@ export const TldrawImage = memo(function TldrawImage(props: TldrawImageProps) {
 
 	return (
 		<div ref={setContainer} style={{ position: 'relative', width: '100%', height: '100%' }}>
-			{url && <img src={url} style={{ width: '100%', height: '100%' }} />}
+			{url && (
+				<img
```

Comment:
we could ban `img` tags without a referrer policy with this `no-restricted-syntax` rule: `JSXOpeningElement[name.name=img]:not(:has(JSXAttribute[name.name=referrerPolicy]))`

Reply:
yeah that was added in already :)